### PR TITLE
build(solution) update to net core 2.2

### PR DESCRIPTION
**Pull request by fabiankoziolek**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/igroup-opole/yesterday-api/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```
Update from net core 2.1 to 2.2.

### CODE REVIEW

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
```
Comment (rafalschmidt97) [inline YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: You only updated target. Try to do the same with packages (like you did with YesterdayApiTest.csproj)
<PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="2.1.0" />

File: YesterdayApi/Core/Users/User.cs [hunk 2]
```diff
@@ -1,21 +1,20 @@
-using System;
-using System.Collections.Generic;
-using YesterdayApi.Core.Users.Posts.Comments;
-using YesterdayApi.Core.Users.Posts.Reactions;
-using YesterdayApi.Core.Users.Profiles;
-using YesterdayApi.UserIdentity.Data.Core.Users;
-
```
Comment (rafalschmidt97) [inline YesterdayApi/Core/Users/User.cs]: Wrong pr 🙆‍♂️

**Comment by rafalschmidt97** [comment]
Travis build could not complete build

`
$ dotnet restore YesterdayApi
/usr/share/dotnet/sdk/2.1.503/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 2.2.  Either target .NET Core 2.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.2. [/home/travis/build/igroup-opole/yesterday-api/YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]
/usr/share/dotnet/sdk/2.1.503/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 2.2.  Either target .NET Core 2.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.2. [/home/travis/build/igroup-opole/yesterday-api/YesterdayApi/YesterdayApi.csproj]
The command "dotnet restore YesterdayApi" failed and exited with 1 during .
`

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj (hunk 1 above)
Comment (fabiankoziolek) [reply YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: The packages are working and up to date, i ran the project :)

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```
Comment (AlexTorianyk) [inline YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: Please make sure to update all of the packages to 2.2 versions in UserIdentity and Test projects.

File: YesterdayApi/Core/Users/User.cs (hunk 2 above)
Comment (fabiankoziolek) [reply YesterdayApi/Core/Users/User.cs]: Yeah, i know, but it was shouting at me.

Comment (fabiankoziolek) [reply YesterdayApi/Core/Users/User.cs]: The gap was created by the previous PR

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj (hunk 3 above)
Comment (fabiankoziolek) [reply YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: They are, read the full PR :)

**Comment by rafalschmidt97** [comment]
> Travis build could not complete build
> 
> ```
> $ dotnet restore YesterdayApi
> /usr/share/dotnet/sdk/2.1.503/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 2.2.  Either target .NET Core 2.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.2. [/home/travis/build/igroup-opole/yesterday-api/YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]
> /usr/share/dotnet/sdk/2.1.503/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(137,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 2.2.  Either target .NET Core 2.1 or lower, or use a version of the .NET SDK that supports .NET Core 2.2. [/home/travis/build/igroup-opole/yesterday-api/YesterdayApi/YesterdayApi.csproj]
> The command "dotnet restore YesterdayApi" failed and exited with 1 during .
> ```

That is my fault. Can you Fabian edit travis to 2.2 with another commit?
Just change this file: https://github.com/igroup-opole/yesterday-api/blob/master/.travis.yml

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj (hunk 1 above)
Comment (rafalschmidt97) [reply YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: Great then! If packages are updated then it's fine 🙆‍♂️ Please notice coworkers that they NEED to update dotnet to 2.2 😀

File: YesterdayApi/Core/Users/User.cs (hunk 2 above)
Comment (rafalschmidt97) [reply YesterdayApi/Core/Users/User.cs]: Can you create another one and our check style everywhere else? It's sad that linting solution doesn't recognise it as an error.

File: YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
-    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="2.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.1.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.1.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.0" />
```
Comment (AlexTorianyk) [inline YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: Great! Please don't be so toxic xd

Comment (rafalschmidt97) [reply YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj]: Thanks @AlexTorianyk 🙆‍♂️

**Comment by rafalschmidt97** [comment]
Nice! @fabiankoziolek can you squash this two commits? I don't have privileges for that:
style(config) remove empty line at the end -> bugfix(config) add missing version property

Or even better squash everything 🙆‍♂️

**Comment by rafalschmidt97** [comment]
Next time try to keep your squash commit simpler - remove for example my bugfixes messages 🙆‍♂️

```

build(solution) update to net core 2.2


bugfix(config) add missing version property

- if every other projects have specified version this one also should has
ci(app) update travis to dotnet 2.2

- it require workaround with other dist (we should keep that in mind in next update)

https://travis-ci.community/t/dotnet-core-2-2/1216/7

style(config) remove empty line

- github editor add empty line at the end which is weird
```

Good job @fabiankoziolek  🚀

File: YesterdayApi/Core/Users/User.cs [hunk 5]
```diff
@@ -1,21 +1,20 @@
...
-  }
+using System;
+using System.Collections.Generic;
+using YesterdayApi.Core.Users.Posts.Comments;
+using YesterdayApi.Core.Users.Posts.Reactions;
+using YesterdayApi.Core.Users.Profiles;
+using YesterdayApi.UserIdentity.Data.Core.Users;
+
+namespace YesterdayApi.Core.Users
+{
```
Comment (rafalschmidt97) [inline YesterdayApi/Core/Users/User.cs]: Changes here are sad but

>  if someone else did mistake then you also can 😫

**Pull request merged**